codex-rs/exec/src/event_processor_with_human_output.rs

```diff
@@ -278,6 +278,7 @@ impl EventProcessor for EventProcessorWithHumanOutput {
                 command,
                 cwd,
                 parsed_cmd: _,
+                ..
```

Comment:
let's just add the extra field and assign to `_` like `parsed_cmd` so the reader immediately knows what is being ignored.

That or we delete `parsed_cmd: _,`. It's weird to use both `_` and `..`.